MercurySQL/gensql/table.py

```diff
@@ -295,26 +302,40 @@ def struct(
         skipError = skipError and force
 
         for name, type_ in columns.items():
+            default_value = None
+            
+
+            # 支持 type_ 为 [str, "默认值"] 的写法
```

Comment:
I suggest that to use `tuple` instead of `list`, since there's already a usage:
```py
tb['col_name'] = str, "hello"
```
The code is as below
https://github.com/BernieHuang2008/MercurySQL/blob/ffc6e463bec44a94546b2438ab5c5474a5698d4b/MercurySQL/gensql/table.py#L148C1-L152C29

Python will parse the situation above as tuple, so it's recommend to have the same form as the one already implemented in `table.__setitem__()`.

Reply:
@lanbinleo 这块最好还是你来改一下吧，我不知道你在哪调用了这玩意

Reply:
![image](https://github.com/user-attachments/assets/2075c10b-6ac2-41eb-af93-4f02c9402e08)
你看这一段